config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -20,11 +19,11 @@ periodics:
       # we need privileged mode in order to do docker in docker
       securityContext:
         privileged: true
-    annotations:
-      testgrid-dashboards: sig-testing-kind
-      testgrid-tab-name: ci unit test
-      testgrid-alert-email: bentheelder@google.com
-      description: kind CI unit test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: ci unit test
```

Comment:
ah, this collides with the postsubmit. we really don't need a periodic unit test for this repo anyhow.